common/config/config.go

```diff
@@ -388,6 +388,9 @@ type (
 	SQL struct {
 		// DBConnector is the SQL driver connector to use. Using this will bypass the string based data source.
 		DBConnector *driver.Connector
+		// DriverName is the name of the SQL driver to be used. This is only relevant with postgres DBConnector.
```

Comment:
If you could extend this to work with MySQL as well I'd appreciate it. We may as well support it in both as it's only a few lines of change.

After that: I've no other comments and am happy to approve